src/crypto/index.ts

```diff
@@ -1832,6 +1832,7 @@ export class Crypto extends TypedEventEmitter<CryptoEvent, CryptoEventHandlerMap
         this.outgoingRoomKeyRequestManager.stop();
         this.deviceList.stop();
         this.dehydrationManager.stop();
+        this.backupManager.stop();
```

Comment:
The changes in this file do not match the PR title

Reply:
https://github.com/matrix-org/matrix-js-sdk/pull/3553